module-2/backend/week_three.md

```diff
@@ -9,17 +9,80 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 
 ### Questions
 
-1. What are the 3 main features in an ERD?
+1. What are the 3 main features in an ERD? ERD is a good tool to visualize the schema of the database.
+
+It shows what columns/attributes there are in tables/models. More importantly, it shows primary keys and foreign keys and therefore is a good indicator of one-to-one and/or one-to-many relationships. Finally, it demonstrates joint tables which stand for many-to-many relationsihps. 
```

Comment:
Think this is an outdated version of the questions. We generally on Friday to make sure that they're consistent with material we've covered. Be sure to pull down the latest version in the future! 👍